Lib/ufo2ft/makeotfParts.py

```diff
@@ -153,11 +167,12 @@ def setupFile_featureTables(self):
         in a different way if desired.
         """
 
-        import tempfile
-        from fontTools.feaLib.builder import addOpenTypeFeatures
-        from fontTools.ttLib import TTFont
+        if self.mtiFeaFiles is not None:
+            for tag, feapath in self.mtiFeaFiles.iteritems():
```

Comment:
`dict.iteritems()` has been removed in Python 3. You should use `dict.items()` if you want to make this compatible with both 2 and 3. If the dict is not so huge, it doesn't make a big of difference if you use a list instead of an iterator.